<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AboutSplit</class>
 <widget class="QDialog" name="AboutSplit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>704</width>
    <height>451</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>10</y>
     <width>611</width>
     <height>321</height>
    </rect>
   </property>
   <column>
    <property name="text">
     <string>ID Клиента</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ФИО</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Фамилия</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Отчество</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID Аренда</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>Лицензия</string>
    </property>
   </column>
   <column>
    <property name="text">
     <string>ID Сплит</string>
    </property>
   </column>
  </widget>
  <widget class="QLineEdit" name="fam">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>380</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Фамилия</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idRent">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>380</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>ID Аренда (0, если нет)</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="license">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>380</y>
     <width>113</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Лицензия</string>
   </property>
  </widget>
  <widget class="QPushButton" name="addClient">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>380</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>ADD</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_2">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>420</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>CLOSE</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>31</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Обн</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>350</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Добавить</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="name">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>380</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Имя</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="otch">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>380</y>
     <width>71</width>
     <height>24</height>
    </rect>
   </property>
   <property name="placeholderText">
    <string>Отчество</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton_4">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>350</y>
     <width>80</width>
     <height>24</height>
    </rect>
   </property>
   <property name="text">
    <string>Список</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
